swagger/v1/component_schemas/MultipleParticipantDeclarationsResponse.yml

```diff
@@ -0,0 +1,26 @@
+description: "A list of participant declarations"
+type: object
+required:
+  - data
+properties:
+  data:
+    type: array
+    items:
+      $ref: "#/components/schemas/ParticipantDeclarations"
+    example:
+      - id: db3a7848-7308-4879-942a-c4a70ced400a
+        type: "participant-declaration"
+        attributes:
+          participant_id: db3a7848-7308-4879-942a-c4a70ced400a
```

Comment:
I think we should make those id-s different - so someone reading the docs does not think they are the same thing.